cmd/utilities.go

```diff
@@ -117,12 +116,7 @@ func UpdateSitemap(cfg *config.Config, commandLine *FlagFields) error {
 	} else {
 		scroll = &sitemap.ElasticScroll{}
 	}
-	var store sitemap.FileStore
```

Comment:
As discussed - I don't think this particular bit of code is triggering the restart as I can't see how the code would be triggered.